cli/defenseclaw/commands/cmd_setup.py

```
@@ -2810,12 +3255,20 @@ def _rotate_token_transaction(
                         "safely stopped; token B was preserved on disk."
                     ) from primary_error
 
+            restore_failed = False
+            try:
+                _rotate_token_restore_hook_snapshots_locked(hook_snapshots)
+                _rotate_token_verify_restored_hook_snapshots_locked(hook_snapshots)
+            except BaseException:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `BaseException`

(BLE001)